canal/canal_test.go

```diff
@@ -61,8 +65,6 @@ func (s *canalTestSuite) SetUpSuite(c *C) {
 	s.execute(c, "DELETE FROM test.canal_test")
 	s.execute(c, "INSERT INTO test.canal_test (content, name) VALUES (?, ?), (?, ?), (?, ?)", "1", "a", `\0\ndsfasdf`, "b", "", "c")
 
-	s.execute(c, "SET GLOBAL binlog_format = 'ROW'")
```

Comment:
why remove this?

Reply:
Because my account dosen't have super privilege to set this global variable. And I think it's ok if mysql is configured to use ROW format already, like the travis config.